docs/source/notes/numerical_accuracy.rst

```diff
@@ -51,6 +51,40 @@ datatype. E.g.:
     a.norm() # produces tensor(inf)
     a.double().norm() # produces tensor(1.4142e+20, dtype=torch.float64), representable in fp32
 
+.. _Linear Algebra Stability:
+
+Linear algebra (``torch.linalg``)
+---------------------------------
+
+Non-finite values
+^^^^^^^^^^^^^^^^^
+
+The external libraries (backends) that ``torch.linalg`` uses provide no guarantees on their behaviour
+when the inputs have non-finite values like ``inf`` or ``NaN``. As such, neither does PyTorch.
+The operations may return a tensor with non-finite values, or raise an exception, or even segfault.
+
+Consider using :func:`torch.isfinite` before calling these functions if your inputs may contain these values.
```

Comment:
This seems good, might be nice to elaborate slightly on what the user would do with the results of this call